src/pages/ProAmmPool/PositionListItem.tsx

```diff
@@ -294,37 +308,36 @@ function PositionListItem({ stakedLayout, farmAvailable, positionDetails, refe }
             </ButtonPrimary>
           ) : (
             <ButtonGroup>
-              {removed || !!stakedLiquidity ? (
-                <ButtonOutlined disabled padding="8px" style={{ flex: 1 }}>
-                  <Text width="max-content" fontSize="14px">
-                    <Trans>Remove Liquidity</Trans>
-                  </Text>
-                </ButtonOutlined>
-              ) : farmAvailable ? (
-                <MouseoverTooltip
-                  text={t`You need to withdraw your deposited liquidity position from the Farm first`}
-                  placement="top"
-                >
-                  <ButtonOutlined
-                    padding="0"
-                    style={{
-                      width: '100%',
-                      color: theme.disableText,
-                      border: `1px solid ${theme.disableText}`,
+              {reasonToDisableRemoveLiquidity ? (
+                <MouseoverTooltip text={reasonToDisableRemoveLiquidity} placement="top">
+                  <Flex
+                    // this flex looks like redundant
+                    // but without this, the cursor will be default
+                    // as we put pointerEvents=none on the button
+                    sx={{
                       cursor: 'not-allowed',
+                      width: '100%',
                     }}
                   >
-                    <Text width="max-content" fontSize="14px" padding="8px">
-                      <Trans>Remove Liquidity</Trans>
-                    </Text>
-                  </ButtonOutlined>
+                    <ButtonOutlined
+                      style={{
+                        padding: '8px',
+                        width: '100%',
+                        pointerEvents: 'none',
```

Comment:
should we update it directly on Button component? https://github.com/react-component/tooltip/issues/18#issuecomment-650864750

Reply:
I don't think we should as we can't interact with the buttons using pointerEvents none. Some buttons have (i) with tooltip and hovering (i) won't trigger tooltip as the event is skipped by the css value